### DISCUSSION TITLE
best idea ever

### DISCUSSION THREAD

**Discussion by worthant** [Imtjl/.github#3] [Announcements]
@Imtjl/developers today @Zerumi and me discovered, that you can create submodules on github

#### What is this? Basically, it is hyper-link to your own repository, that looks like that:
![image](https://user-images.githubusercontent.com/43885024/229159032-d5ddec2c-e85c-4af3-949c-7d1cad12e68c.png)

#### Why?
- avoids code repetition
- easier to contribute (just add this link and you are done, no need to create special structure for IMTJL)
- more handy overall

> Now i highly recommend adding this practice to every place where it can be usefull, such as adding laboratory works and other stuff, related to personal owning

**Comment by Zerumi** [comment]
// todo: make a tutorial.
While cloning repository you must enter this 2 commands:
```git submodule init```
and ```git submodule update```
Source: https://git-scm.com/book/en/v2/Git-Tools-Submodules
